filament/src/PerViewUniforms.cpp

```diff
@@ -214,8 +214,14 @@ void PerViewUniforms::prepareDirectionalLight(
     }
 }
 
-void PerViewUniforms::prepareAmbientLight(FIndirectLight const& ibl,
-        float intensity, float exposure) noexcept {
+void PerViewUniforms::prepareIblLight(const IblOptions& options) noexcept {
+    auto& s = mPerViewUb.edit();
+    s.iblTechnique = static_cast<uint32_t>(options.iblTechnique);
+    s.iblCenter = options.iblCenter;
+    s.iblHalfExtents = options.iblHalfExtents;
+}
+
+void PerViewUniforms::prepareAmbientLight(FIndirectLight const& ibl, float intensity, float exposure) noexcept {
```

Comment:
nit: could have preserved the formatting for clearer diff. If they change `prepareAmbientLight` we will conflict during bumping to newer Filament.

Reply:
note: if you'd act on my comment here, you don't have to recompile the filament libs in https://github.com/shapr3d/shapr3d/pull/9453 (because you only update the formatting of the code) and you will have to update version.info anyway so I guess you can still revert to the original formatting

Reply:
Fixed in https://github.com/shapr3d/filament/pull/49/commits/5ddc987d1553ecd566f5f139c244416be16a2211